po/extra/pt.po

```diff
@@ -1,107 +1,118 @@
 # Portuguese translations for extra package.
 # Copyright (C) 2019 THE extra'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the extra package.
-# Automatically generated, 2019.
+# André Barata <andretiagob@protonmail.com>, 2020.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: extra\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2019-03-26 23:12+0100\n"
-"PO-Revision-Date: 2019-03-26 13:52+0100\n"
-"Last-Translator: Automatically generated\n"
+"PO-Revision-Date: 2020-08-26 15:18+0100\n"
+"Last-Translator: André Barata <andretiagob@protonmail.com>\n"
 "Language-Team: none\n"
 "Language: pt\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.4.1\n"
 
 #: data/com.github.gijsgoudzwaard.image-optimizer.appdata.xml.in:6
 #: data/com.github.gijsgoudzwaard.image-optimizer.desktop.in:3
 #: data/com.github.gijsgoudzwaard.image-optimizer.desktop.in:4
 msgid "Image Optimizer"
-msgstr ""
+msgstr "Otimizador de Imagem"
```

Comment:
Since `Image Optimizer` is the name of the app it shouldn't be translated.

Reply:
Done